atomix/cluster/src/main/java/io/atomix/raft/partition/impl/RaftPartitionServer.java

```diff
@@ -198,6 +199,7 @@ public CompletableFuture<Void> flushLog() {
     return server.flushLog();
   }
 
+  @VisibleForTesting
```

Comment:
This method is called ONLY from the `ZeebeIT` test. Ideally we would remove it, but I figured it wouldn't hurt to add the annotation to make it clearer that ideally we shouldn't use it in production code.